src/guards/jwt-auth.guard.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, CanActivate, ExecutionContext } from '@nestjs/common';
+import { Reflector } from '@nestjs/core';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class JwtAuthGuard implements CanActivate {
+  constructor(private reflector: Reflector, private jwtService: JwtService) {}
+
+  canActivate(context: ExecutionContext): boolean {
```

Comment:
async-await và viết exception cụ thể hơn về UnauthorizedException không trả về true false

Reply:
return true is acceptable, just not return false